cephfs/preview_canary_test.go

```diff
@@ -0,0 +1,11 @@
+// +build ceph_preview
```

Comment:
I would make the opposite test, that it does fail, when `ceph_preview` is _not_ set, because compile a file is the default. (If you would have a typo in the `+build`, it would still fail.)

Reply:
If it wasn't clear from the patch title and PR description I mean to simply drop that entire patch from the PR once it's served its purpose.  I don't understand " a typo in the +build, it would still fail."  Do you mean like if you typed "+bbuild" or something?

Reply:
Yes... or maybe `// +build` doesn't work, only `//+ build` ? (I didn't check the other files...)

Reply:
For example, if there would be no empty line following this line, the constrain would be ineffective and the test would fail, even if the tag is not set. (As I just found out one minute ago.)

Reply:
I pushed a new version that adds a 2nd file that has the `!ceph_preview` constraint. Inspecting the CI failure should show only the test failure for the contstriant set but not the 2nd one.
FWIW, I tested both scenarios locally - with and without `NO_PREVIEW=true` and each test failure appeared when I expected it to. If you're OK with it... I will remove the patch from the PR.

Reply:
Of course, go ahead!